src/ui/buttons/buttons.jsx

```diff
@@ -106,9 +106,11 @@ export function Buttons(props : ButtonsProps) : ElementNode {
         nonce, components, onShippingChange, onShippingAddressChange, onShippingOptionsChange, personalization, userIDToken, content, flow, experiment, applePaySupport,
         supportsPopups, supportedNativeBrowser, experience } = normalizeButtonProps(props);
     const { custom, layout, shape, tagline } = style;
- 
+
     const inlineExperience = experience === EXPERIENCE.INLINE && custom && custom.label && custom.label.length !== 0;
 
+    const fundingSourcesWithInlinePaymentFields = [ FUNDING.EPS ];
```

Comment:
can we use the `inlineExperience` above or some similar function so that it is easy to maintain and ramp up/down.